content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -91,73 +103,89 @@ For example:
+On large clusters, retrieving the collection of some resource types may result in very large responses that can impact the server and client. For instance, a cluster may have tens of thousands of pods, each of which is 1-2KiB of encoded JSON. Retrieving all pods across all namespaces may result in a very large response (10-20MB) and consume a large amount of server resources. The kube-apiserver supports the ability to break a single large collection request into many smaller chunks while preserving the consistency of the total request. Each chunk can be returned sequentially; this chunking reduces the size of each HTTP response message and also allows user-oriented clients to display results incrementally to improve responsiveness. Splitting retrieval over multiple HTTP responses means that if a client needs to retry a fetch, only the missing portion needs to be retransmitted.
 
-To retrieve a single list in chunks, two new parameters `limit` and `continue` are supported on collection requests and a new field `continue` is returned from all list operations in the list `metadata` field. A client should specify the maximum results they wish to receive in each chunk with `limit` and the server will return up to `limit` resources in the result and include a `continue` value if there are more resources in the collection. The client can then pass this `continue` value to the server on the next request to instruct the server to return the next chunk of results. By continuing until the server returns an empty `continue` value the client can consume the full set of results.
+To retrieve a single list in chunks, two parameters `limit` and `continue` are supported on collection requests. When a List request specifies a `limit`, a field `continue` is returned in the `metadata` field. A client should specify the maximum results they wish to receive in each chunk with `limit`; the API server returns up to `limit` resources in the result. If there are more resources in the collection, the API server response also includes a `continue` field. The client can then pass this `continue` value to the API server in the next request, instructing the API server to return the next chunk of results. By continuing until the server returns an empty `continue` value, the client can consume the full set of results.
 
-Like a watch operation, a `continue` token will expire after a short amount of time (by default 5 minutes) and return a `410 Gone` if more results cannot be returned. In this case, the client will need to start from the beginning or omit the `limit` parameter.
+Like a Watch operation, a `continue` token expires after a defined amount of time (5 minutes by default). If more results cannot be returned the API server returns a 410 (Gone) status. On receiving a 410 (Gone) response, the client needs to make a new request (possibly omitting the `limit` parameter, or using a larger `limit`).
```

Comment:
Check about formatting response codes and capitalization?

Reply:
410 is the right code and isn't capitalized. “Gone” is the text that 410 represents, both in English and as commonly returned as the status string by HTTP/1.x servers. However, this is not an actual HTTP/1.1 response; those look like:
```
HTTP/1.1 410 Gone
…
```

See https://tools.ietf.org/html/rfc7231#section-6.5.9 for an example of idiomatic representation in running text.